docs/deploy.md

```diff
@@ -174,30 +232,26 @@ Confirm that all of the servers came up properly (and requested their keys
 correctly):
 
     $ spire verify online
+      # if this fails, it's possible that your ssh-agent might be borked and you need to restart it.
```

Comment:
'borked' isn't a technical term, and while fine in speech, isn't quite right for documentation. may as well use 'broken'